api/controllers/ApplicationTypeController.js

```diff
@@ -296,16 +294,29 @@ module.exports = {
                                                     first_name: account.first_name,
                                                     last_name: account.last_name,
                                                     telephone: account.telephone,
-                                                  mobileNo: account.mobileNo,
-                                                  email: user.email,
+                                                    mobileNo: account.mobileNo,
+                                                    email: user.email,
                                                     confirm_email: user.email,
                                                     has_email: true
                                                 }).then(function(){
-                                                    if (req.session.appType ==2) {
-                                                        return res.redirect('/business-document-quantity?pk_campaign=Premium-Service&pk_kwd=Premium');
-                                                    }
-                                                    else if (req.session.appType ==3) {
-                                                        return res.redirect('/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff');
+                                                    if (
+                                                        req.session.appType == 2
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/business-document-quantity?pk_campaign=Premium-Service&pk_kwd=Premium'
+                                                        );
+                                                    } else if (
+                                                        req.session.appType == 3
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff'
+                                                        );
+                                                    } else if (
+                                                        req.session.appType === 4
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/check-document-eligibility'
+                                                        );
```

Comment:
I'd be pretty tempted to throw in another else here and throw an error - currently it's a black whole if appType isn't one of 2, 3, or 4 - this probably never happens but good to code defensively. (Of course throwing an Error here might crash the app so need to handle it properly, maybe just a log statement....)

Reply:
Yeah, that's a fair point. I'll make a note to update this.

- [ ] Add else for if appType is not 1,2,3 or 4